Tennis-scheduler/src/main/java/com/tennis/tennisscheduler/dtos/PersonDto.java

```diff
@@ -17,4 +17,5 @@ public class PersonDto {
     public Date birthday;
     public Address address;
     public String password;
+    public String confirmPassword;
```

Comment:
Where is this field used?

Reply:
This field is added because of form on FE, but we didn't provide that check on BE because we have it on FE.

Reply:
Ok, if it's not needed here please delete it.